### PR TITLE
Pr/627

### PR DESCRIPTION
**Pull request by Kizuno18**
added a delay before ask for otp, it solved all problems

### CODE REVIEW

**Comment by vermaysha** [comment]
What a problem ?

**Review by ugurgungezerler — APPROVED** [review]

**Comment by guiireal** [comment]
I didn't understand, without this delay is it not sending to you?

**Comment by Kizuno18** [comment]
Yes, exactly, without delay it doesn't work for me.
When I put some delay it worked 🔥

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "changelog:last": "conventional-changelog -p angular -r 2",
     "changelog:preview": "conventional-changelog -p angular -u",
     "changelog:update": "conventional-changelog -p angular -i CHANGELOG.md -s -r 0",
-    "example": "node --inspect -r ts-node/register Example/example.ts",
+    "example": "node --inspect -r ts-node/register Example/example.ts --mobile",
```
Comment (purpshell) [inline package.json]:
```suggestion
    "example": "node --inspect -r ts-node/register Example/example.ts",
    "example:mobile": "node --inspect -r ts-node/register Example/example.ts --mobile",
```

**Pull request merged**